test/built-ins/DataView/prototype/getBigInt64/toindex-byteoffset.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype.getbigint64
+description: >
+  ToIndex conversions on byteOffset
+info: |
+  DataView.prototype.getBigInt64 ( byteOffset [ , littleEndian ] )
+
+  1. Let v be the this value.
+  2. If littleEndian is not present, let littleEndian be undefined.
+  3. Return ? GetViewValue(v, byteOffset, littleEndian, "Int64").
+
+  24.3.1.1 GetViewValue ( view, requestIndex, isLittleEndian, type )
+
+  ...
+  4. Let getIndex be ? ToIndex(requestIndex).
+  ...
+includes: [typeCoercion.js]
+features: [DataView, ArrayBuffer, DataView.prototype.setUint8, BigInt, Symbol, Symbol.toPrimitive]
+---*/
+
+var buffer = new ArrayBuffer(12);
+var sample = new DataView(buffer, 0);
+
+sample.setUint8(0, 39);
+sample.setUint8(1, 2);
+sample.setUint8(2, 6);
+sample.setUint8(3, 2);
+sample.setUint8(4, 128);
+sample.setUint8(5, 0);
+sample.setUint8(6, 128);
+sample.setUint8(7, 1);
+sample.setUint8(8, 127);
+sample.setUint8(9, 0);
+sample.setUint8(10, 1);
+sample.setUint8(11, 2);
+
+testCoercibleToIndexZero(function (x) {
+  assert.sameValue(sample.getBigInt64(x), 2810815725239828481n);
```

Comment:
Verified these and following tests by converting these numbers to hex in Python and seeing that they make sense. If you wanted to make these tests a little cleaner and more obvious to debug, you could use hex syntax instead.